src/index.js

```diff
@@ -15,6 +15,10 @@ require('./index.scss').toString();
  */
 import ToolboxIcon from '../assets/icon.svg';
 import SettingsIcon from '../assets/settings-icon.svg';
+import IconAlignLeft from '../assets/IconAlignLeft.svg';
+import IconAlignCenter from '../assets/IconAlignCenter.svg';
+import IconAlignRight from '../assets/IconAlignRight.svg';
```

Comment:
```suggestion
import AlignLeftIcon from '../assets/align-left-icon.svg';
import AlignCenterIcon from '../assets/align-center-icon.svg';
import AlignRightIcon from '../assets/align-right-icon.svg';
```

Not a big deal, but it would help preserve the existing naming convention style.

Reply:
If i commit here, without changing the file names themselves, i assume that will break things - not sure on the best way to resolve, commit then pull the repo then push a new change with filename changes and the code changes?

Reply:
Pull using `git pull origin master`, commit changes and then push it. Everything happens on your fork. Once the PR is merged, all the changes in your fork's master would then be merged to the master branch of this repo.